core/overdrive.py

```diff
@@ -902,6 +902,18 @@ def link(self, page, rel):
         ),
     }
 
+    # A mapping of the overdrive format name to end sample content type
+    # Overdrive samples are not DRM protected so the links should be
+    # stored as the end sample content type
+    sample_format_to_content_type = {
+        "ebook-overdrive": "text/html",
+        "audiobook-wma": "application/octet-stream",
```

Comment:
It doesn't really matter, since the mobile apps can't play it anyway, but does wma here stand for [windows media audio](https://en.wikipedia.org/wiki/Windows_Media_Audio)? If it does, its probably better to use the media type listed on that wikipedia page `audio/x-ms-wma`.

Reply:
Yes, that is what it represents. Will change the format.